pcdsdevices/epics/attenuator.py

```diff
@@ -491,8 +491,7 @@ class does not include any filters.
 
     def __init__(self, prefix, *, name=None, read_attrs=None, ioc="",
                  stage_setting=0, **kwargs):
-        prefix = prefix + ":ATT:COM"
-        self._filter_prefix = prefix + ":ATT"
+        self._filter_prefix = prefix.rstrip(':COM')
```

Comment:
Can we meet halfway on this one? It was set up as it was before because the ioc literally enforces the `ATT:COM` part of the prefix. I agree that passing in just `XPP` is a bad move, but can we make it `XPP:ATT`?

Excerpt from IOC template file:
record(calcout, $(HUTCH):ATT:10:_SET_IN) {
  field(OUT, "$(DEVSTATEBASE):ATT:10:GO_IN PP")
  field(CALC, "(A>>B)&1")
  field(DOPT, "Use CALC")
  field(OOPT, "When Non-zero")
  field(B, "9") 
}

record(longout, $(HUTCH):ATT:COM:_C_GO) {
  field(DESC, "BitVector-Ceiling")
}

Reply:
Sounds good.